api/projects/models.py

```diff
@@ -152,6 +154,10 @@ def trigger_environments_v2_migration(self) -> None:
     def write_to_dynamo(self):
         write_environments_to_dynamodb.delay(kwargs={"project_id": self.id})
 
+    @hook(AFTER_DELETE)
+    def clean_up_dynamo(self):
+        DynamoProjectMetadata(self.id).delete()
```

Comment:
Only environment delete creates a task because it needs to delete a ton of objects (rest of them are sync). I'd prefer just making a call to dynamo instead of Postgres to create the task. What do you think?

Reply:
Yeah, I'm torn I quite like the idea of `models` never interacting directly with dynamo as a pattern, but I don't think I feel strongly enough about it here.